### PR TITLE
OSDOCS-4381:Adding support for k8s nmstate on OpenStack

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12

Issue:

https://issues.redhat.com/browse/OSDOCS-4381
Link to docs preview:

https://51941--docspreview.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.html
QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by qiowang721** [comment]
It works for me. Just one thing want to confirm, should we also update the example output of the installation using the CLI?

**Comment by skrthomas** [comment]
Thanks both :) @qiowang721 Yes, good catch. I created this example in 4.10, so I think this version needs updating in 4.11 too. I'll have to open a separate PR to do that.

File: modules/k8s-nmstate-deploying-nmstate-CLI.adoc [hunk 1]
```diff
@@ -99,5 +99,5 @@ oc get clusterserviceversion -n openshift-nmstate \
 [source, terminal]
 ----
 Name                                             Phase
-kubernetes-nmstate-operator.4.10.0-202203120157   Succeeded
+kubernetes-nmstate-operator.4.1.0-202210210157   Succeeded
```
Comment (qiowang721) [inline modules/k8s-nmstate-deploying-nmstate-CLI.adoc:102]: maybe you mean kubernetes-nmstate-operator.4.12.0-202210210157? ^ ^

Comment (skrthomas) [reply modules/k8s-nmstate-deploying-nmstate-CLI.adoc]: oh yes, I did mean to put 12. I think I deleted the '0'  from '10' and then realized the date also needed changing and then didn't add the 2.

**Review by creydr — COMMENTED** [review]
/lgtm

File: networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ The Kubernetes NMState Operator provides a Kubernetes API for performing state-d
 
 [IMPORTANT]
 ====
-Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, and VMware vSphere installations.
+Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, VMWare vSphere, and OpenStack installations.
```
Comment (creydr) [inline networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:14]:
```suggestion
Red Hat supports the Kubernetes NMState Operator in production environments on bare-metal, IBM Power, IBM Z, LinuxONE, VMware vSphere, and OpenStack installations.
```
According to their website it is written "VMware" (https://www.vmware.com/company.html)

**Review by creydr — COMMENTED** [review]
/

File: networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc (hunk 2 above)
Comment (skrthomas) [reply networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc:14]: iNtErEsTiNg...updating!

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-needed
/label peer-review-in-progress

**Review by tmalove — COMMENTED** [review]
/remove-label peer-review-in-progress
/label peer-review-done
/LGTM

**Pull request merged**